diff --git a/node_modules/@aws-sdk/lib-storage/dist-es/bytelength.js b/node_modules/@aws-sdk/lib-storage/dist-es/bytelength.js
index 3358945..ef9758b 100644
--- a/node_modules/@aws-sdk/lib-storage/dist-es/bytelength.js
+++ b/node_modules/@aws-sdk/lib-storage/dist-es/bytelength.js
@@ -1,4 +1,4 @@
-import { ClientDefaultValues } from "./runtimeConfig";
+import { ClientDefaultValues } from "./runtimeConfig.browser";
 export var byteLength = function (input) {
     if (input === null || input === undefined)
         return 0;
